bpf/lib/nodeport.h

```diff
@@ -1260,6 +1260,12 @@ static __always_inline int nodeport_svc_lb6(struct __ctx_buff *ctx,
 	ret = lb6_local(get_ct_map6(tuple), ctx, l3_off, l4_off,
 			key, tuple, svc, &ct_state_svc,
 			nodeport_xlate6(svc, tuple), ext_err, 0);
+#ifdef ENABLE_L7_LB
```

Comment:
just a question for my own knowledge (feel free to resolve): Why do we need a new mechanism to redirect to the proxy? Why isn't the existing one sufficient?

Reply:
The old one pushes all traffic into the proxy, but this is not what we want given only traffic should be pushed into a proxy iff there was a "backend" which is a local proxy, and if there is not, we push all traffic via xdp/bpf datapath off the node again. Meaning, these are two different mechanisms. Reusing the same flag with the very limited bits we have in the svc value is also tricky since proxy port overlaps with session affinity timeout, and we would like to use the latter feature too. Hope that helps.